spec.html

```diff
@@ -130,11 +130,12 @@ <h1>HostResizeArrayBuffer ( _buffer_, _newByteLength_ )</h1>
     <h1>The ArrayBuffer Constructor</h1>
 
     <emu-clause id="sec-arraybuffer-length">
-      <h1>ArrayBuffer ( _length_ [ , _options_ ] )</h1>
+      <h1>ArrayBuffer ( _length_<ins>[ , _options_ ]</ins> )</h1>
       <p>When the `ArrayBuffer` function is called with argument _length_<ins> and optional argument _options_</ins>, the following steps are taken:</p>
       <emu-alg>
         1. If NewTarget is *undefined*, throw a *TypeError* exception.
         1. Let _byteLength_ be ? ToIndex(_length_).
+        1. <ins>If _options_ is not present, set _options_ to *undefined*.</ins>
```

Comment:
I don't think this step, or the similar one in SAB, is necessary. Not present optional parameters are already undefined.